docs/adding_packages/dependencies.md

```diff
@@ -5,90 +5,24 @@ from handling "vendored" dependencies to what versions should be used.
 
 <!-- toc -->
 ## Contents
-
-  * [List Dependencies](#list-dependencies)
-    * [Optional Requirements](#optional-requirements)
-    * [Build Requirements](#build-requirements)
-  * [Accessing Dependencies](#accessing-dependencies)
-    * [Handling Requirement's Options](#handling-requirements-options)
-    * [Verifying Dependency's Version](#verifying-dependencys-version)
-    * [Passing Requirement's info to `build()`](#passing-requirements-info-to-build)
-    * [Overriding the provided properties from the consumer](#overriding-the-provided-properties-from-the-consumer)
+  * [Handling Requirement's Options](#handling-requirements-options)
   * [Adherence to Build Service](#adherence-to-build-service)
     * [Version Ranges](#version-ranges)
-      * [Adding Version Ranges](#adding-version-ranges)
+    * [Adding Version Ranges](#adding-version-ranges)
   * [Handling "internal" dependencies](#handling-internal-dependencies)<!-- endToc -->
 
-## List Dependencies
```

Comment:
Requirements and logic involving it is well covered by Conan docs:

- https://docs.conan.io/2/tutorial/creating_packages/add_dependencies_to_packages.html
- https://docs.conan.io/2/reference/conanfile/methods/requirements.html